starter-code/index.js

```diff
@@ -1,17 +1,58 @@
 class SortedList {
-  constructor() {}
+  constructor(items, length) {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length++;
```

Comment:
la propiedad this.length debería estar mirando el length the this.items, y de esta manera no tendrías q ir sumando uno en cada addItem